services/121-service/src/payments/fsp-integration/intersolve-visa/services/__snapshots__/intersolve-visa.service.spec.ts.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing
+
+exports[`IntersolveVisaService retrieveAndUpdateAllWalletsAndCards should throw when 10 IntersolveVisaApiErrors occur 1`] = `"10 IntersolveVisaApiErrors occurred while retrieving and updating wallets and cards, for parent tokenCodes: , , , , , , , , , . Aborting."`;
```

Comment:
The parent token codes are missing in this test. I'm assuming this is by accident. If not then I'd add a comment in the test to make it less surprising

Personally I wouldn't have used a snapshot for this. I can imagine that's why you missed the parent token codes missing (assuming it was unintentional).